core/src/main/scala/anorm/MayErr.scala

```diff
@@ -1,36 +0,0 @@
-/*
- * Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com>
- */
-package anorm
-
-// TODO: Remove
-
-private[anorm] case class MayErr[+E, +A](toEither: Either[E, A]) {
```

Comment:
Did you just remove this to simplify the implementation by using `Either` instead?